Troubleshooting.md

```diff
@@ -137,41 +137,3 @@ This will remove the finalizers on the namespace.
 NAMESPACE=<namespace>
 kubectl get namespace $NAMESPACE -o json | sed 's/"kubernetes"//' | kubectl replace --raw "/api/v1/namespaces/$NAMESPACE/finalize" -f -
 ```
-
-## Resolve Conflicts error while deploying VPC CNI managed add-on
```

Comment:
Removed this since this `OVERWRITE` is the default for nearly all modules - let me know if we still want to retain this though